src/test/java/RESTAssuredTests/APICallsTrailingslashUpdate.java

```diff
@@ -0,0 +1,55 @@
+package RESTAssuredTests;
+
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import com.oracle.webservices.internal.api.message.ContentType;
+
+import io.restassured.RestAssured;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.filter.log.RequestLoggingFilter;
+import io.restassured.filter.log.ResponseLoggingFilter;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+
+import static io.restassured.RestAssured.*;
+import static org.hamcrest.Matchers.*;
+
+import org.junit.BeforeClass;
+
+public class APICallsTrailingslashUpdate {
+	
+	
+	
+	@Test
+	public void testAPIcallswithoutlashes()
+	{
+		given()            
+        .header("API-KEY", "***")
+        .header("content-type", "application/json")
+        .header("Authorization", "Bearer token")
+        .when()
+        .get("https://chpl.ahrqstg.org/rest/announcements/")
+        .then()
+        .log()
+        .all();
+        //Assert.assertEquals(statusCode, 200, "Correct status code returned");		
+		given().get("https://chpl.ahrqstg.org/rest/announcements/").then().statusCode(200).log().all();
```

Comment:
Do you want to also test announcements without a / ? Looks like all the other calls below test both with and without /.